debian/control.d/control.buster.in

```diff
@@ -4,4 +4,4 @@ Depends: libavahi-client3 (>= 0.6.16), libavahi-common3 (>= 0.6.16), libc6 (>= 2
     python (<< 2.8), python (>= 2.7~), python:any (<< 2.8), python:any (>= 2.7~), libboost-python1.62.0 | libboost-python1.67.0, libboost-system1.62.0 | libboost-system1.67.0,
     libboost-thread1.62.0 | libboost-thread1.67.0,
     bc, cgroup-tools, libtk-img (>= 1.13), procps, psmisc, python-tk, python-numpy, python-xlib,
-    python-gtkglext1, python-configobj, python-zmq, python-protobuf (>= 2.4.1), python-avahi, python-simplejson, python-pyftpdlib, tclreadline, tcl8.6, tk8.6, uuid-runtime, yapps2
+    python-gtkglext1, python-configobj, python-zmq, python-protobuf (>= 2.4.1), python-avahi, python-simplejson, python-pyftpdlib, tclreadline, tcl8.6, tk8.6, uuid-runtime, yapps2, libck0
```

Comment:
There's no `libck0` package in Buster.

Reply:
Don't know what is going on with Debian at present.

This situation is the same for quite a few packages at present, if it is in sid, why don't they just backport it, instead of leaving Buster with no package?

I am actually using sid presently so did not even think to check if Buster was covered.

Does your `/etc/apt/sources.list.d` in the dockers have a `machinekit.list` file?
I am very tempted to just build a libck0 package for Buster and host it in the repo, rather than have the stupid situation where people running the most up to date distro, cannot use the ck features.

Reply:
Yes, this is dumb.  I'll ask around the Debian community at some point.

The mk-cross-builder images do configure the MK apt repository.  If you add that package, let me know and I'll update the images.

Did you see my comment in #221, `libck` is also not built for Jessie i386?